### PR TITLE
Add feature flag doc

### PR DESCRIPTION
**Pull request by catherineluse**
This PR https://github.com/rancher/docs/pull/1817 has merge conflicts so this PR follows up on it, including the same changes.

This PR addresses this issue: https://github.com/rancher/docs/issues/1773

### CODE REVIEW

**Comment by catherineluse** [comment]
I'm going to respond to Denise's feedback on https://github.com/rancher/docs/pull/1817 on this PR

File: content/rancher/v2.x/en/admin-settings/_index.md [hunk 1]
```diff
@@ -53,3 +53,9 @@ The information that Rancher uses to provision [RKE clusters]({{< baseurl >}}/ra
 Rancher Kubernetes Metadata contains Kubernetes version information which Rancher uses to provision [RKE clusters]({{< baseurl >}}/rancher/v2.x/en/cluster-provisioning/rke-clusters/).
 
 For more information on how metadata works and how to configure metadata config, see [Rancher Kubernetes Metadata]({{< baseurl >}}/rancher/v2.x/en/admin-settings/rke-metadata/).
+
+## Enabling Unsupported Features
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/admin-settings/_index.md]: We should update experimental in this section.

Comment (catherineluse) [reply content/rancher/v2.x/en/admin-settings/_index.md]: updated

File: content/rancher/v2.x/en/admin-settings/_index.md [hunk 2]
```diff
@@ -53,3 +53,9 @@ The information that Rancher uses to provision [RKE clusters]({{< baseurl >}}/ra
 Rancher Kubernetes Metadata contains Kubernetes version information which Rancher uses to provision [RKE clusters]({{< baseurl >}}/rancher/v2.x/en/cluster-provisioning/rke-clusters/).
 
 For more information on how metadata works and how to configure metadata config, see [Rancher Kubernetes Metadata]({{< baseurl >}}/rancher/v2.x/en/admin-settings/rke-metadata/).
+
+## Enabling Experimental Features
+
+_Available as of v2.3.0_
+
+Rancher includes some features that are unsupported and disabled by default. Feature flags were introduced to allow you to try these features. For more information, refer to the section about [feature flags.]({{<baseurl>}}/rancher/v2.x/en/admin-settings/feature-flags)
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/admin-settings/_index.md]: we should update `unsupported` here to `experimental` as well.

Comment (catherineluse) [reply content/rancher/v2.x/en/admin-settings/_index.md]: updated

**Pull request merged**